packages/shared/src/styles/colors.ts

```diff
@@ -1,20 +1,23 @@
-export type ColorName =
-  | 'burger'
-  | 'blueCheese'
-  | 'avocado'
-  | 'lettuce'
-  | 'cheese'
-  | 'bun'
-  | 'ketchup'
-  | 'bacon'
-  | 'cabbage'
-  | 'onion'
-  | 'water'
-  | 'salt'
-  | 'pepper'
-  | 'twitter'
-  | 'whatsapp'
-  | 'facebook';
+export enum Colors {
+  Burger = 'burger',
+  BlueCheese = 'blueCheese',
+  Avocado = 'avocado',
+  Lettuce = 'lettuce',
+  Cheese = 'cheese',
+  Bun = 'bun',
+  Ketchup = 'ketchup',
+  Bacon = 'bacon',
+  Cabbage = 'cabbage',
+  Onion = 'onion',
+  Water = 'water',
+  Salt = 'salt',
+  Pepper = 'pepper',
+  Twitter = 'twitter',
+  Whatsapp = 'whatsapp',
+  Facebook = 'facebook',
+}
+
+export type ColorName = Uncapitalize<keyof typeof Colors>;
```

Comment:
We don't need this anymore I believe, on line 23 we can just `Record<Colors, Color>`. We can even retain the `ColorName` on the enum part.

Reply:
Removed, thanks for pointing that out 🙇